aten/src/TH/generic/THBlas.cpp

```diff
@@ -380,6 +383,39 @@ void THBlas_(gemm)(
     return;
   }
 #endif
+
+#ifdef USE_FBGEMM
+#if defined(TH_REAL_IS_LONG)
```

Comment:
nit: might be cleaner to combine these two, e.g. `#if defined(USE_FBGEMM) && defined(TH_REAL_IS_LONG)` , to reduce nested if defs.

Reply:
Updated. Thanks!